foundry.toml

```diff
@@ -21,4 +21,4 @@ fuzz_max_local_rejects = 4096
 runs           = 10     # The number of calls to make in the invariant tests
 depth          = 1000   # The number of times to run the invariant tests
 call_override  = false  # Override calls
-fail_on_revert = true   # Fail the test if the contract reverts
+fail_on_revert = false   # Fail the test if the contract reverts
```

Comment:
Is this intentional? 🤔 What's the actual consequence of this flag? It seems like a test won't fail if there's an uncaught/unexpected revert